.github/workflows/global-release.yaml

```diff
@@ -0,0 +1,242 @@
+          workflow: release
+          token: ${{ secrets.HYDRO_SERVING_TOKEN }}
+          repo: ${{ github.repository_owner }}/${{ matrix.service_repo_name }}
+          inputs: '{"version":"${{ env.VERSION }}", "release":"global", "grpc_version":"${{ env.VERSION }}"}'
+       
+  Release_serving:
+    needs: [version, Release_services]
+    runs-on: ubuntu-latest
+    
+    env:
+      VERSION: ${{ needs.version.outputs.version }}
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - env:
+          IMAGE: serving-gateway
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+          
+      - env:
+          IMAGE: serving-manager
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+          
+      - env:
+          IMAGE: hydro-root-cause
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+      
+      - env:
+          IMAGE: sonar
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+        
+      - env:
+          IMAGE: stat
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+      
+      - env:
+          IMAGE: hydro-serving-ui
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+          
+      - env:
+          IMAGE: hydro-visualization
+        name: Bump ${{ env.IMAGE }} version in chart
+        uses: ./.github/actions/release-action
+        with:
+          release: global
+          registry_url: ${{ env.DOCKER_REGISTRY_URL }}
+          service_image_name: ${{ env.IMAGE }}
+          tag: ${{ env.VERSION }}
+          
+      - name: Setup helm
+        run: |
+          curl -LO https://git.io/get_helm.sh
```

Comment:
Now we can use helm3 - $ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3